sample-apps/dotnet/README.md

```diff
@@ -0,0 +1,37 @@
+# DotNet Demo Sample App Updating Guide
+
+## Introduction:
+
+The dotnet sample app is used to perform E2E testing on cloudwatch, cloudwatch operator and adot repository. If any changes need to be made on the demo sample app, the following steps should be taken.
+
+## EKS Use Case: Uploading to ECR
+
+### Build the sample app locally:
+run `docker-compose build` to generate two docker images: `dotnetsampleapp/frontend-service` and `dotnetsampleapp/remote-service`
+
+### Steps to update image:
```

Comment:
Call out that the actual sample app artifacts used in the test are uploaded using the workflows created in this PR. The steps you listed here should only be used for testing in personal accounts.

Reply:
Done